cmd/provision.go

```diff
@@ -98,6 +98,15 @@ See https://roots.io/trellis/docs/remote-server-setup/#provision for more detail
 Provision the production environment:
 
   $ trellis provision production
+  
+Options:
```

Comment:
I'd prefer to keep the options at the bottom for now just because it's how every command does it so its consistent. Want to just make the change below?

Reply:
Makes sense. I have done so. I forget if I need to resubmit the pull request or what. I think it updates automatically.